RadixWallet/Core/FeaturePrelude/AddressView/AddressDetails+View.swift

```diff
@@ -231,11 +170,40 @@ private extension AddressDetails.ViewState {
 
 		return result
 	}
+}
 
-	static func enlarged(address: LedgerIdentifiable.Address) -> AttributedString {
+// MARK: - Enlarged view
+private extension AddressDetails.View {
+	var enlargedView: some SwiftUI.View {
+		Group {
```

Comment:
What is the purpose of the Group?

Reply:
probably left over from old implementation, removing